src/senaite/sync/browser/views.py

```diff
@@ -258,6 +272,16 @@ def update_object_with_data(self, obj, data, domain):
             if isinstance(value, dict) and value.get("uid"):
                 # dereference the referenced object
                 value = self.dereference_object(value.get("uid"), uidmap)
+            elif type(value) in (list, tuple):
+                for i, item in enumerate(value):
```

Comment:
Why do you need to enumerate in this loop? `i` is not used...

Reply:
I don't remember at all :)